src/branch.rs

```diff
@@ -45,6 +45,10 @@ where
         self.0.len()
     }
 
+    pub fn is_empty(&self) -> bool {
+        self.0.is_empty()
+    }
+
```

Comment:
Why is this needed? It will always return false.
Please do a separate PR for code changes.

Reply:
It's needed because you added `len` method, and it's good practice have always both.

> It is good custom to have both methods, because for some data structures, asking about the length will be a 
> costly operation, whereas .is_empty() can usually answer in constant time. Also it used to lead to false positives
> on the len_zero lint – currently that lint will ignore such entities.

If it always return false we can just make it false.

It's part of this PR since we're checking the warnings and the codestyle before commit code, and it will be likely part of the CI.
If you want to have a separate PR, it means I have to address those nits before the CI, it's fine by me.

Reply:
Yeah i would make a separate PR since it adds API that is not in the released version.

Reply:
Filed https://github.com/dusk-network/microkelvin/pull/6